.github/workflows/build.yml

```diff
@@ -26,36 +26,40 @@ jobs:
           distribution: 'zulu'
           java-version: ${{ matrix.java }}
       # step 3
+      - name: "Print maven version"
+        run: ./mvnw -version
+      # step 4
       - name: "Build with Maven"
         run: |
           if [ "${{ matrix.java }}" == "8" ]; then
             ./mvnw -T 4C clean test -Dcheckstyle.skip=false -Dlicense.skip=false -e -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn;
           elif [ "${{ matrix.java }}" == "17" ]; then
             ./mvnw -T 4C clean test -Dcheckstyle.skip=true  -Dlicense.skip=true  -e -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn;
           fi
-      # step 4
+      # step 5
       - name: "Codecov"
         if: matrix.java == '17'
         uses: codecov/codecov-action@v2.1.0
 
   # job 2: Test on 'arm64v8/ubuntu' OS.
   build_arm64-binary:
-     runs-on: ubuntu-latest
-     if: ${{ github.event_name == 'push' && (github.ref_name == 'develop' || github.ref_name == 'snapshot' || github.ref_name == '2.x') }}
-     strategy:
-       fail-fast: false
-     steps:
-       # step 1
-       - uses: actions/checkout@v2
-       # step 2
-       - name: Set up QEMU
-         id: qemu
-         uses: docker/setup-qemu-action@v1
-       # step 3
-       - name: Build arm-binary
-         run: |
-           docker run --rm -v ${{ github.workspace }}:/ws:rw --workdir=/ws \
-             arm64v8/ubuntu:20.04 \
-             bash -exc 'apt-get update -y && \
-             apt-get install maven -y && \
-             mvn -Prelease-seata -Dmaven.test.skip=true clean install -U'
+    runs-on: ubuntu-latest
```

Comment:
need to limit event_name to push. The build_arm64 pipeline takes a long time.

Reply:
I temporarily removed it and now withdraw it.